### PR TITLE
Dwesterveld/add dummy data

### PR DESCRIPTION
**Pull request by djwester**

### CODE REVIEW

File: test/components/d2l-rubric-criteria-group.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+
+<html>
+	<head>
+		<title>d2l-rubric-criteria-group test</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+		<script src="https://s.brightspace.com/lib/siren-parser/6.5.0/siren-parser.js"></script>
+		<script src="../../../webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (alogan519) [inline test/components/d2l-rubric-criteria-group.html]: tab spacing still looks off in this file

Comment (alogan519) [reply test/components/d2l-rubric-criteria-group.html]: hmm looks fine when i pick view file

Comment (djwester) [reply test/components/d2l-rubric-criteria-group.html]: yeah it seems to be something in the display on github - they all look fine for me locally...

Comment (djwester) [reply test/components/d2l-rubric-criteria-group.html]: do we usually use tabs or spaces? @alogan519

Comment (alogan519) [reply test/components/d2l-rubric-criteria-group.html]: tabs

**Pull request merged**